mllib/src/main/scala/org/apache/spark/ml/util/SchemaUtils.scala

```diff
@@ -61,6 +61,20 @@ private[spark] object SchemaUtils {
   }
 
   /**
+    * Check whether the given schema contains a column of the numeric data type.
```

Comment:
indent 1 space less (can update IntelliJ code style settings: Editor -> Code Style -> Scala -> ScalaDoc tab -> uncheck Use scaladoc indent for leading asterisk